mypy/test/testcmdline.py

```diff
@@ -78,6 +79,7 @@ def test_python_cmdline(testcase: DataDrivenTestCase) -> None:
                                            path))
     else:
         out = normalize_error_messages(out)
+        out.append('== Return code: {}'.format(result))
```

Comment:
Hm, this added a lot of noise to a lot of tests. Perhaps only print the return code when it's not equal to `not len(out)`?